src/platform/platform_winuser.c

```diff
@@ -544,6 +531,101 @@ CxPlatUtf8ToWideChar(
     return HRESULT_FROM_WIN32(Error);
 }
 
+_IRQL_requires_max_(PASSIVE_LEVEL)
+_Must_inspect_result_
+_Success_(return != FALSE)
+BOOLEAN
+CxPlatProcessorGroupInfo(
+    _In_ LOGICAL_PROCESSOR_RELATIONSHIP Relationship,
+    _Outptr_ _At_(*Buffer, __drv_allocatesMem(Mem)) _Pre_defensive_
+        PSYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX* Buffer,
+    _Out_ PDWORD BufferLength
```

Comment:
You never seem to use this output. Remove it?

Reply:
We use it in CxPlatProcessorInfoInit